vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -236,107 +227,175 @@ An `env.json` file to configure the application URLs and client credentials must
 
 Instructions to export a bundle, including how to configure `env.json`, can be found in the [Export and Publish a Bundle tutorial](../../tutorials/create/pb/export-bundle-from-application.md).
 
-## Profile Management
-To manage and switch between different ent configurations, use `ent profile`. 
 
-To switch between different Entando Applications, even if they are in different clusters, `ent profile` uses Kubernetes contexts, kubeconfig files, and/or custom commands (refer to `ent profile first-use-readme`).
+#### ECR Wrapper
+Entando provides a series of `ent ecr` commands for interaction with [the Entando Component Repository](../../docs/compose/ecr-overview.md) (ECR). The following commands apply specifically and only to bundles.
```

Comment:
above in line 72(could not leave comment there), 'ECR' appears without the full name which may confuse beginners. It would be good to have the full name there too since this document is not necessarily used in a linear fashion.